packages/code_excerpt_updater/lib/code_excerpt_updater_cli.dart

```diff
@@ -9,11 +9,10 @@ import 'package:logging/logging.dart';
 
 import 'src/code_excerpt_updater.dart';
 import 'src/logger.dart';
-import 'src/nullable.dart';
 
-const _commandName = 'code_excerpt_updater';
```

Comment:
Do we need to add types here? Adds a lot of noise IMHO.

Reply:
I guess I somehow got used to adding types to top-level variables. I'll remove these changes as they're not related to the intention.